docs/crashstorage/elasticsearch.rst

```diff
@@ -127,41 +127,86 @@ contains data about each field from Elasticsearch that can be manipulated.
 The list is managed in code in ``socorro/external/es/super_search_fields.py``
 as a dict of ``name`` -> ``properties``.
 
+
+Names
+-----
+
 The name of a field is exposed in the Super Search API. This must be unique.
 
+
+Properties
+----------
+
 Here is an explanation of each properties of a field:
 
-+----------------------+---------------------------------------------------------+
-| Parameter            | Description                                             |
-+======================+=========================================================+
-| in_database_name     | Name of the field in the database and in Elasticsearch. |
-+----------------------+---------------------------------------------------------+
-| namespace            | Namespace of the field. Separated with dots.            |
-+----------------------+---------------------------------------------------------+
-| description          | Description of the field, for admins only.              |
-+----------------------+---------------------------------------------------------+
-| query_type           | Defines operators that can be used in Super Search.     |
-|                      | See details below.                                      |
-+----------------------+---------------------------------------------------------+
-| data_validation_type | Defines the validation done on values passed to         |
-|                      | filers of this field in Super Search.                   |
-+----------------------+---------------------------------------------------------+
-| permissions_needed   | Permissions needed for a user to access this field.     |
-+----------------------+---------------------------------------------------------+
-| form_field_choices   | Choices offered for filters of that field in the        |
-|                      | Super Search form.                                      |
-+----------------------+---------------------------------------------------------+
-| is_exposed           | Is this field exposed as a filter?                      |
-+----------------------+---------------------------------------------------------+
-| is_returned          | Is this field returned in results?                      |
-+----------------------+---------------------------------------------------------+
-| has_full_version     | Does this field have a full version in Elasticsearch?   |
-|                      | Enable only if you use a multitype field in the         |
-|                      | storage mapping.                                        |
-+----------------------+---------------------------------------------------------+
-| storage_mapping      | Mapping that is used in Elasticsearch for this field.   |
-|                      | See Elasticsearch documentation for more info.          |
-+----------------------+---------------------------------------------------------+
+**name**
+    The name of the field.
```

Comment:
I wanted to add links and examples and other bits to this data, so I switched it from a table to a definition list.